cloudformation/ELK_Stack_Multi_AZ_in_Private_VPC.json

```diff
@@ -276,10 +293,20 @@
                     "Fn::Base64": {
                         "Fn::Join": [ "\n", [
                             "#!/bin/bash -v",
-
+                            "echo \"127.0.0.1 `hostname`\" >> /etc/hosts",
                             "wget -O - http://packages.elasticsearch.org/GPG-KEY-elasticsearch | apt-key add -",
-                            "echo \"deb http://packages.elasticsearch.org/logstash/1.4/debian stable main\" > /etc/apt/sources.list.d/logstash.list",
-                            "echo \"deb http://packages.elasticsearch.org/elasticsearch/1.3/debian stable main\" > /etc/apt/sources.list.d/elasticsearch.list",
+
+                            { "Fn::If": [
+                                "Kibana3",
+                                { "Fn::Join": [ "\n", [
+                                    "echo \"deb http://packages.elasticsearch.org/logstash/1.4/debian stable main\" > /etc/apt/sources.list.d/logstash.list",
+                                    "echo \"deb http://packages.elasticsearch.org/elasticsearch/1.3/debian stable main\" > /etc/apt/sources.list.d/elasticsearch.list"
+                                ]]},
+                                { "Fn::Join": [ "\n", [
+                                    "echo \"deb http://packages.elasticsearch.org/logstash/1.4/debian stable main\" > /etc/apt/sources.list.d/logstash.list",
+                                    "echo \"deb http://packages.elasticsearch.org/elasticsearch/1.4/debian stable main\" > /etc/apt/sources.list.d/elasticsearch.list"
+                                ]]}
+                            ]},
```

Comment:
I was thinking of putting this in mappings rather than a conditional, which would be better suited.

I actually started to place the whole lot of this JSON file to be generated via sparkle formation (https://github.com/ajohnstone/elk-stack/pull/2).still lots to do on that. Maintaining a JSON file is very messy.